api/interservice/deployment/automate_deployment.proto

```diff
@@ -124,6 +124,7 @@ message GenerateAdminTokenRequest {
 
 message GenerateAdminTokenResponse {
 	string api_token = 1;
```

Comment:
Could we change this to `token_value` or is that causing a lot of undue work? Every time I will see this I will be wondering, "Wait, there's an ID of a token next, so this is... a whole token? no, so what then...?"

Reply:
it's only being used in one place as far as I can tell so I can make that change